pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+python_files = test/*.py
```

Comment:
there should be no need to have a `test/__init__.py` file, see https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code